docs/src/modules/utils/getJsxPreview.js

```diff
@@ -10,7 +10,7 @@ export default function getJsxPreview(code) {
    * `  );\n}` or `;\n}`
    */
   let jsx = code.match(
-    /export default .*(?:\n {2}return \(\n|\n {2}return )(?: {4}<(?:div|Stack|Box).*?>\n)?(.*?)(\n {4}<\/(?:div|Stack|Box)>)?(\n {2}\);\n}|;\n})/s,
+    /export default .*(?:\n {2}return \(\n|\n {2}return )(?: {4}<(?:div|Stack|Box)(?:[^\n]*>|.*\n\s+>)?\n)?(.*?)(\n {4}<\/(?:div|Stack|Box)>)?(\n {2}\);\n}|;\n})/s,
```

Comment:
First case is for single-line JSXOpeningTag:
```js
<Box disabled>
```
Second case for multi-line JSXOpeningTag:
```js
<Box
  sx={{}}
>
```

Note that this relies on the way prettier formats code.